### PR TITLE
feat(BUX-222): exposing mock consts for tests

### PR DESCRIPTION
**Pull request by kuba-4chain**

### CODE REVIEW

File: broadcast/broadcast-client-mock/mock-consts/mock_consts.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	MockedTimestamp                 = "2023-09-05T17:03:49.537230128Z"
+	MockedTimestampSecondary        = "2023-09-05T17:05:29.736256927Z"
+	MockedTxResponseStatus          = 200
+	MockedTxResponseTitle           = "OK"
+	MockedTxStatus                  = "SEEN_ON_NETWORK"
+	MockedTxBlockHash               = "123123"
+	MockedTxBlockHashSecondary      = "123123"
+	MockedTxBlockHeight             = 123456
+	MockedTxBlockHeightSecondary    = 123456
+	MockedTxExtraInfo               = ""
```
Comment (dorzepowski) [inline broadcast/broadcast-client-mock/mock-consts/mock_consts.go:22]: Do we need this `Mocked` prefix in each and every const name?

File: broadcast/broadcast-client-mock/mock-consts/mock_consts.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package mock_consts
```
Comment (dorzepowski) [inline broadcast/broadcast-client-mock/mock-consts/mock_consts.go:1]: Maybe `stubs` or `fixtures` instead?

File: broadcast/broadcast-client-mock/mock-consts/mock_consts.go (hunk 1 above)
Comment (kuba-4chain) [reply broadcast/broadcast-client-mock/mock-consts/mock_consts.go]: Done

File: broadcast/broadcast-client-mock/mock-consts/mock_consts.go (hunk 2 above)
Comment (kuba-4chain) [reply broadcast/broadcast-client-mock/mock-consts/mock_consts.go]: Okay, I switched to `fixtures`

**Review by dorzepowski — APPROVED** [review]

**Pull request merged**